modules/trusted-firmware-m/CMakeLists.txt

```diff
@@ -26,6 +26,9 @@ if (CONFIG_BUILD_WITH_TFM)
     endif()
   endif()
 
+  # Treat any warning as error
```

Comment:
shouldnt this be only enabled when `CONFIG_COMPILER_WARNINGS_AS_ERRORS=y`?

Reply:
yep ! Fixed it.